benchmarks/imports.py

```diff
@@ -2,9 +2,6 @@
 
 
 class ImportBench(BaseBench):
-    repeat = 1
-    timeout = 12000
```

Comment:
We should reconsider repeat here - default timeout is 300s and default repeat value is 50 - its probably not enough time to import cats and dogs that much time.

Reply:
:) though gc has chance on finishing